api/v1/controllers/samples.js

```diff
@@ -302,7 +305,10 @@ module.exports = {
         .then((samples) => {
           samples.forEach((sample) => {
             if (!sample.isFailed) {
-              publisher.publishSample(sample, subHelper.model);
+              if (!featureToggles
+                .isFeatureEnabled(sampleStoreConstants.featureName)) {
+                publisher.publishSample(sample, subHelper.model);
+              }
             }
```

Comment:
We are doing it to see if moving the publishSample to cache can introduce some kind of a throttle on the realtime events.